Addons/properties-templates/naviox.properties

```diff
@@ -6,6 +6,13 @@ storePasswordAsHex=true
 initialPasswordForAdmin=admin
 # There are many more password options in Configuration module of XavaPro (http://www.openxava.org/xavapro)
 
+#Amazon s3 details
+bucketName=bucketName
+accessKey=accessKey
+secretKey=secretKey
+region=region
+bucketPrefixFolderName=bucketPrefixFolderName

```

Comment:
Question: Is this extra property required? For simplicity sake store everything from root folder?